lapixdl/base.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+import os
+
+
+class FileTypeError(RuntimeError):
+    pass
+
+
+def basename(
```

Comment:
Isn't it better to just use the Pathlib

Reply:
I really don't like the Pathlib pattern for join (`/`), using the `os` functions seems clearer to me